.travis-ci.sh

```diff
@@ -15,15 +15,7 @@ elif [[ $TASK = 'htmlproofer-external' ]]; then
   bundle exec jekyll build
   bundle exec htmlproofer $HTMLPROOFER_OPTIONS --external_only
 elif [[ $TASK = 'eip-validator' ]]; then
-  BAD_FILES="$(ls EIPS | egrep -v "eip-[0-9]+.md|eip-20-token-standard.md")" || true
-  if [[ ! -z $BAD_FILES ]]; then
-    echo "Files found with invalid names:"
-    echo $BAD_FILES
-    exit 1
-  fi
-
-  FILES="$(ls EIPS/*.md | egrep "eip-[0-9]+.md")"
-  bundle exec eip_validator $FILES
+  eipv EIPS/ --ignore=title_max_length,missing_discussions_to --skip=eip-20-token-standard.md
```

Comment:
Please note that the validator is currently not ensuring that titles are less than the max specified length and that each EIP has a `discussions-to` field. This is due to the number of non-conforming EIPs. I think formally increasing the max title length to support all current EIPs and then turning on the check is acceptable. With regards to the `discussions-to` field, there are some options:
1. In a separate PR, go in and create discussions threads for every EIP missing one
2. Do 1) in this PR
3. Forget historic EIPs with no `discussions-to` and only check new ones

I prefer `1`.

Reply:
I prefer 1 or 2.  Anything that gets the system into a more standardized state so we can reliably use the bot going forward and don't need to maintain code to support malformed ancient EIPs.

Reply:
Also, prefer 1 or 2

Reply:
I'll follow the `1` path.